vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthSignUpSignInSelectionFragment.kt

```diff
@@ -60,27 +60,27 @@ class FtueAuthSignUpSignInSelectionFragment @Inject constructor() : AbstractSSOF
             ServerType.MatrixOrg -> {
                 views.loginSignupSigninServerIcon.setImageResource(R.drawable.ic_logo_matrix_org)
                 views.loginSignupSigninServerIcon.isVisible = true
-                views.loginSignupSigninTitle.text = getString(R.string.login_connect_to, state.homeServerUrlFromUser.toReducedUrl())
+                views.loginSignupSigninTitle.text = getString(R.string.login_connect_to, state.selectedHomeserver.userFacingUrl.toReducedUrl())
                 views.loginSignupSigninText.text = getString(R.string.login_server_matrix_org_text)
             }
             ServerType.EMS       -> {
                 views.loginSignupSigninServerIcon.setImageResource(R.drawable.ic_logo_element_matrix_services)
                 views.loginSignupSigninServerIcon.isVisible = true
                 views.loginSignupSigninTitle.text = getString(R.string.login_connect_to_modular)
-                views.loginSignupSigninText.text = state.homeServerUrlFromUser.toReducedUrl()
+                views.loginSignupSigninText.text = state.selectedHomeserver.userFacingUrl.toReducedUrl()
             }
             ServerType.Other     -> {
                 views.loginSignupSigninServerIcon.isVisible = false
                 views.loginSignupSigninTitle.text = getString(R.string.login_server_other_title)
```

Comment:
We don't need setting the icon in `ServerType.Other` case?
`views.loginSignupSigninServerIcon.setImageResource(R.drawable.ic_logo_element_matrix_services)` or maybe set it to null?

Reply:
for the `other` case we're hiding the icon with `views.loginSignupSigninServerIcon.isVisible = false`, this icon is the one under the element logo

| OTHER | MATRIX.ORG
| --- | --- |
![other-flow](https://user-images.githubusercontent.com/1848238/162425096-940dc43d-432c-4ac6-8e90-750ce91ca004.gif)|![matrix-icon](https://user-images.githubusercontent.com/1848238/162425490-b9e5b1b5-707e-4d96-99ec-89037d34a6e1.gif)

Reply:
Thanks, for explaining